docs/docs/self-hosting/deploy/loadbalancing-example/loadbalancing-example.mdx

```diff
@@ -62,7 +62,7 @@ export ZITADEL_MASTERKEY="$(tr -dc A-Za-z0-9 </dev/urandom | head -c 32)"
 docker compose up --detach
 ```
 
-Make `127.0.0.1` available at `my.domain`. For example, this can be achived with an entry `127.0.1.1 my.domain` in the `/etc/hosts` file.
+Make `127.0.0.1` available at `my.domain`. For example, this can be achived with an entry `127.0.0.1 my.domain` in the `/etc/hosts` file.
```

Comment:
please help to correct typo here also
achived -> achieved

Reply:
Typo fixed.